tests/fixture/tmpnet/node.go

```diff
@@ -104,16 +104,16 @@ func NewEphemeralNode(flags FlagsMap) *Node {
 }
 
 // Initializes the specified number of nodes.
-func NewNodes(count int) ([]*Node, error) {
```

Comment:
nit: very perl-esque naming... I think something like `OrPanic` or `OrFatal` is probably more in the style of Go